[2023-06-13T06:51:40.290+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry.add_entry_to_db_task manual__2023-06-13T06:51:39.014149+00:00 [queued]>
[2023-06-13T06:51:40.297+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry.add_entry_to_db_task manual__2023-06-13T06:51:39.014149+00:00 [queued]>
[2023-06-13T06:51:40.297+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-13T06:51:40.298+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-13T06:51:40.299+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-13T06:51:40.306+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): add_entry_to_db_task> on 2023-06-13 06:51:39.014149+00:00
[2023-06-13T06:51:40.311+0000] {standard_task_runner.py:55} INFO - Started process 781 to run task
[2023-06-13T06:51:40.314+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'add_entry', 'add_entry_to_db_task', 'manual__2023-06-13T06:51:39.014149+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/AssignmentDag.py', '--cfg-path', '/tmp/tmpb50x905m']
[2023-06-13T06:51:40.316+0000] {standard_task_runner.py:83} INFO - Job 57: Subtask add_entry_to_db_task
[2023-06-13T06:51:40.368+0000] {task_command.py:389} INFO - Running <TaskInstance: add_entry.add_entry_to_db_task manual__2023-06-13T06:51:39.014149+00:00 [running]> on host 4f6c6e07547c
[2023-06-13T06:51:40.417+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=add_entry
AIRFLOW_CTX_TASK_ID=add_entry_to_db_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T06:51:39.014149+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-13T06:51:39.014149+00:00
[2023-06-13T06:51:40.426+0000] {base.py:73} INFO - Using connection ID 'postgres_connection' for task execution.
[2023-06-13T06:51:40.430+0000] {sql.py:364} INFO - Running statement: INSERT INTO assignment_table (execution_time) VALUES ('2023-06-13 06:51:40.419017'), parameters: None
[2023-06-13T06:51:40.432+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-13T06:51:40.433+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-13T06:51:40.441+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=add_entry, task_id=add_entry_to_db_task, execution_date=20230613T065139, start_date=20230613T065140, end_date=20230613T065140
[2023-06-13T06:51:40.487+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-13T06:51:40.501+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
